spec/js/ko-components/view-models/orifice-calculator-spec.coffee

```diff
@@ -44,18 +44,42 @@ describe 'orifice-calculator-viewmodel-spec', ->
       expect(viewModel.pipeID().length).toEqual 6
 
     it 'should have default value of "1.939\'\' XS, Sch 80, Sch 80S"', ->
-      expect(viewModel.selectedPipeID()).toEqual '1.939\'\' XS, Sch 80, Sch 80S'
+      expect(viewModel.selectedPipeID()).toEqual OPL.OrificeCalculator.Config.Dictionaries.PipeID.one_nine_inch
 
   describe 'operatingPressure', ->
     itBehavesLikeMandatoryField viewModel.operatingPressure
     itBehavesLikeIntegerField viewModel.operatingPressure
 
+    describe 'when pressure is high', ->
+      it 'should be invalid at > 200', ->
+        viewModel.operatingPressure(300)
+        expect(viewModel.operatingPressure.isValid()).toEqual false
+        viewModel.operatingPressure(1000)
+        expect(viewModel.operatingPressure.isValid()).toEqual false
+
+      it 'should display a warning message at > 200', ->
+        viewModel.operatingPressure(201)
+        expect(viewModel.operatingPressure.error()).toEqual OPL.OrificeCalculator.Config.Dictionaries.Messages.warning
```

Comment:
1- Don't rely on observable validations for error message. The way I see it an error is something that stops the user from completing a task, which is not the case. Rather use a pureComputed to decide which feedback to give to the user
2- `OPL.OrificeCalculator.Config.Dictionaries.Messages.warning` is very vague. If I show you this variable name without any usage background you have no clue of what will be translated to. Please add at least a reference to the operating pressure. Something along `OPL.OrificeCalculator.Config.Dictionaries.Messages.operatingPressureWarning`